docs/running-a-node/configuration.md

```diff
@@ -182,14 +182,27 @@ enable_voting = true
 #### node.enable_voting
 As of V18.0, newly setup nodes have voting disabled by default. In order to participate in network consensus, this value must be updated to enable voting and the node restarted.
 
-
 ---
 
 #### node.vote_minimum
 As of V18.0, nodes with weight delegated to them under this value in their config will not vote, regardless of the `enable_voting` flag setting. In order for a node to vote, this value must be lower than the weight delegated to the representative account setup on the node.
 
 ---
 
+#### node.enable_control
```

Comment:
`enable_control` is part of the rpc config and is at a top level not under `[node]`